app/templates/buyers/dashboard.html

```diff
@@ -127,7 +127,7 @@
             <div class="padding-bottom-small">
                 <p><a href="{{ url_for('.view_brief_responses', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">View responses</a></p>
                 {% if item.status == "closed" %}
-                    <p><a class="award-contract-link" href="{{ url_for('.award_brief', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">Tell us who won this contract</a></p>
+                    <p><a class="award-contract-link" href="{{ url_for('.award_or_cancel_brief', framework_slug=item.frameworkSlug, lot_slug=item.lot, brief_id=item.id) }}">Let suppliers know the outcome</a></p>
```

Comment:
FYI this copy change will mean the functional tests (in `award_requirements.feature`) will fail on preview/staging - the link text and URL are both hardcoded there.